### PR TITLE
DM-2434 Updated "How does the platform support developing agents?" section.

### PR DESCRIPTION
**Pull request by MaciekSoftwareAG**

### CODE REVIEW

File: content/concepts/interfacing-devices-bundle/developing-agents.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ layout: redirect
 
 {{< product-c8y-iot >}} supports agent development on three different levels:
 
-* There are a number of full-featured open source agents and drivers in {{< company-c8y >}}'s [github.com](https://github.com/SoftwareAG/cumulocity-examples) and [mbed.org](http://mbed.org/users/vwochnik/code/) repositories. More information can be found in the "Devices" section of this documentation.
+* There are a number of full-featured open source agents and drivers in {{< company-c8y >}}'s [github.com](https://github.com/SoftwareAG/cumulocity-examples) and [mbed.org](http://mbed.org/users/vwochnik/code/) repositories, e.g. [Cumulocity IoT Device Management Reference Agent](https://github.com/SoftwareAG/cumulocity-devicemanagement-agent) in Python.
```
Comment (rssag) [inline content/concepts/interfacing-devices-bundle/developing-agents.md:10]:
```suggestion
* There are a number of full-featured open source agents and drivers in {{< company-c8y >}}'s [github.com](https://github.com/SoftwareAG/cumulocity-examples) and [mbed.org](http://mbed.org/users/vwochnik/code/) repositories, for example, the [{{< product-c8y-iot >}} Device Management Reference Agent](https://github.com/SoftwareAG/cumulocity-devicemanagement-agent) in Python.
```

**Comment by rssag** [comment]
what are the target versions?

**Review by l2p- — CHANGES_REQUESTED** [review]
cumulocity-examples does not contain a well supported and fully featured device agent. The mbed agent is also long dead. Both of these need to go. Please also add a link to thin-edge.

File: content/concepts/interfacing-devices-bundle/developing-agents.md [hunk 2]
```diff
@@ -7,6 +7,6 @@ layout: redirect
 
 {{< product-c8y-iot >}} supports agent development on three different levels:
 
-* There are a number of full-featured open source agents and drivers in {{< company-c8y >}}'s [github.com](https://github.com/SoftwareAG/cumulocity-examples) and [mbed.org](http://mbed.org/users/vwochnik/code/) repositories. More information can be found in the "Devices" section of this documentation.
+* It supports open source agents and drivers, for example [{{< product-c8y-iot >}} Device Management Reference Agent](https://github.com/SoftwareAG/cumulocity-devicemanagement-agent) or [thin-edge.io](https://thin-edge.github.io/thin-edge.io/)
```
Comment (rssag) [inline content/concepts/interfacing-devices-bundle/developing-agents.md:10]:
```suggestion
* It supports open source agents and drivers, for example [{{< product-c8y-iot >}} Device Management Reference Agent](https://github.com/SoftwareAG/cumulocity-devicemanagement-agent) or [thin-edge.io](https://thin-edge.github.io/thin-edge.io/).
```

**Review by rssag — APPROVED** [review]

**Comment by l2p-** [comment]
> what are the target versions?

10.19

**Review by l2p- — APPROVED** [review]

**Pull request merged**